Index: src/database/init.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>create table VenueManagers\r\n(\r\n    email varchar(50) not null comment 'Stores email id'\r\n        primary key\r\n)\r\n    comment 'Table to store data for Venue Manager';\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/database/init.sql b/src/database/init.sql
--- a/src/database/init.sql	
+++ b/src/database/init.sql	
@@ -2,6 +2,6 @@
 (
     email varchar(50) not null comment 'Stores email id'
         primary key
-)
-    comment 'Table to store data for Venue Manager';
 
+)
+    comment 'Table to store data for Venue Manager';
\ No newline at end of file
